client/package.json

```diff
@@ -32,12 +32,10 @@
     "eject": "react-scripts eject"
   },
   "devDependencies": {
+    "eslint": "^3.19.0",
```

Comment:
I personally don't use the `^` in dependencies, as I have experienced problems with breaking changes multiple times (although the changes should be compatible). We _always_ use strict version pinning.

I don't mean that you *have* to change this, but I just wanted to let you know that this is a potential source of trouble.

Reply:
Thanks, makes totally sense to me.